### PR TITLE
Boy_314's Preonic and XD75 Layouts

### PR DESCRIPTION
**Pull request by Boy-314**

### CODE REVIEW

File: keyboards/preonic/keymaps/boy314/config.h [hunk 1]
```diff
@@ -0,0 +1,44 @@
+#ifndef CONFIG_USER_H
```
Comment (drashna) [inline keyboards/preonic/keymaps/boy314/config.h]: If you could replace the first two lines with just `#pragma once`, and remove the endif at the end?

File: keyboards/preonic/keymaps/boy314/config.h [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+#define CONFIG_USER_H
+
+#define FORCE_NKRO
+
+#include "config_common.h"
```
Comment (drashna) [inline keyboards/preonic/keymaps/boy314/config.h]: This isn't needed, and should be removed.

File: keyboards/preonic/keymaps/boy314/keymap.c [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "preonic.h"
```
Comment (drashna) [inline keyboards/preonic/keymaps/boy314/keymap.c]: Could you replace both of these includes with just `#include QMK_KEYBOARD_H`

File: keyboards/preonic/keymaps/boy314/keymap.c [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+ * | Tab  |   Q  |   W  |   E  |   R  |   T  |   Y  |   U  |   I  |   O  |   P  | Enter|
+ * |------+------+------+------+------+-------------+------+------+------+------+------|
+ * | Caps |   A  |   S  |   D  |   F  |   G  |   H  |   J  |   K  |   L  |   ;  |   '  |
+ * |------+------+------+------+------+------|------+------+------+------+------+------|
+ * | Shift|   Z  |   X  |   C  |   V  |   B  |   N  |   M  |   ,  |   .  |   /  | Shift|
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * | Ctrl |  Del | Alt  | GUI  |Lower |    Space    |Raise | Left | Down |  Up  |Right |
+ * `-----------------------------------------------------------------------------------'
+ */
+[_QWERTY] = {
```
Comment (drashna) [inline keyboards/preonic/keymaps/boy314/keymap.c]: Could you update these so they're using this format:
https://github.com/qmk/qmk_firmware/blob/master/keyboards/preonic/keymaps/default/keymap.c#L53

File: keyboards/xd75/keymaps/boy_314/config.h [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef CONFIG_USER_H
```
Comment (drashna) [inline keyboards/xd75/keymaps/boy_314/config.h]: This file isn't necessary for you, but you could replace all of this with just `#pragma once`

File: keyboards/xd75/keymaps/boy_314/keymap.c [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "xd75.h"
```
Comment (drashna) [inline keyboards/xd75/keymaps/boy_314/keymap.c]: Could you replace this with just `#include QMK_KEYBOARD_H`?

File: keyboards/xd75/keymaps/boy_314/rules.mk [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+ifndef QUANTUM_DIR
```
Comment (drashna) [inline keyboards/xd75/keymaps/boy_314/rules.mk]: This block isn't needed, and is a holdover from the old make system.

If you could remove it?

**Comment by Boy-314** [comment]
@drashna Updated. Please look over the changes and let me know if there's anything else you would like to see changed.

File: keyboards/xd75/keymaps/default/readme.md [hunk 8]
```diff
@@ -1 +1,8 @@
-# The default keymap for xd75, with led controls
+# Boy_314's XD75RE Layout
```
Comment (drashna) [inline keyboards/xd75/keymaps/default/readme.md]: Could you revert this?

File: keyboards/xd75/config.h [hunk 9]
```diff
@@ -85,7 +85,6 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
  * power-up.
  *
  */
-//#define FORCE_NKRO
```
Comment (drashna) [inline keyboards/xd75/config.h]: Could you revert this change?

File: keyboards/xd75/keymaps/default/readme.md (hunk 8 above)
Comment (Boy-314) [reply keyboards/xd75/keymaps/default/readme.md]: Do you want me to replace my entire readme with the default one? Sorry I'm new to this and the formatting is kinda confusing to me.

Comment (Boy-314) [reply keyboards/xd75/keymaps/default/readme.md]: Oh my bad I'm looking at the wrong file.

**Comment by Boy-314** [comment]
@drashna Problems should be fixed now.

**Comment by drashna** [comment]
Awesome, Thanks!
